app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -130,8 +130,13 @@ def available_releases
 
     api :GET, "/activation_keys/:id/product_content", N_("Show content available for an activation key")
     param :id, String, :desc => N_("ID of the activation key"), :required => true
+    param :organization_id, :number, :desc => N_("organization identifier to associate with activation key")
     def product_content
-      content = @activation_key.available_content
+      if params.key?(:organization_id)
+        content = @activation_key.organization == @organization ? @activation_key.available_content : []
+      else
+        content = @activation_key.available_content
```

Comment:
An activation key should already be associated to a single organization, which to me would imply that whats not handling things correctly is this available_content method call.